docs/source/python/filesystems.rst

```diff
@@ -178,11 +178,19 @@ Example how you can read contents from a S3 bucket::
    >>> f.readall()
    b'some data'
 
+   # Note, to resolve the S3 bucket, one can use `resolve_s3_region('my-test-bucket')`
```

Comment:
I would move this to an actual text paragraph instead of "hiding" it somewhat in a comment in a longer example. And maybe first stating that the S3FileSystem needs to be configured with the correct region for the bucket you want to read. And then if you want to do that automatically instead of passing the region manually (as in the example above), explain and show there are two ways you can do that.

Reply:
Indeed, I think that's better. :+1: